nominees/berkely-ucb-particle.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "Berkely UCB Particle and Temperature Sensor UCB Pats",
```

Comment:
@ijeomaemeruwa kindly correct the spelling for the name field to match the [website](https://berkeleyair.com/). It appears to be `Berkeley` not `Berkely`. Please clarify.